pkg/api/v1beta1/dynakube/convert_to.go

```diff
@@ -86,7 +86,7 @@ func (src *DynaKube) toActiveGateSpec(dst *v1beta2.DynaKube) {
 }
 
 func (src *DynaKube) toMovedFields(dst *v1beta2.DynaKube) error {
-	if src.Annotations[AnnotationFeatureMetadataEnrichment] == "false" || src.ClassicFullStackMode() {
```

Comment:
As discussed on Slack, we need to also disable metadata enrichment if there was no injection in `v1beta1`, e.g. only Activegate configured.

Enabling metadata enrichment by default for all namespaces (no `namespaceSelector` configured because it's not used) would conflict with all other Dynakubes that may have a correctly configured `namespaceSelector`

Reply:
done